indexing-service/src/main/java/io/druid/indexing/common/task/AbstractTask.java

```diff
@@ -179,4 +183,24 @@ public int hashCode()
   {
     return toolbox.getTaskActionClient().submit(new LockListAction());
   }
+
+  @Override
+  @JsonProperty
+  public Map<String, Object> getContext()
+  {
+    return context;
+  }
+
+  @Override
+  public <ContextType> ContextType getContextValue(String key)
+  {
+    return context == null ? null : (ContextType) context.get(key);
```

Comment:
I agree there. Can this just be Object?

or alternatively, have some other way of doing better type enforcing?